templates/libvirtd.conf.erb

```diff
@@ -264,7 +264,11 @@ auth_tls = "<%= @auth_tls %>"
 # entirely rather than using empty list to disable these checks
 #
 # By default, no DN's are checked
+<%- if @tls_allowed_dn_joined -%>
+tls_allowed_dn_list = [ <%= @tls_allowed_dn_joined %> ]
```

Comment:
```
igalic@levix ~> puppet apply -e "fail(join(undef, '\",\"'))"
Error: Evaluation Error: Error while evaluating a Function Call, join(): Requires array to work with  at line 1:6 on node levix
```
so we're fine, cuz you have a guard if @tls_allowed_dn_joined anyway.

but i'd add an `else` branch above, which explicitly sets the variable to undef, thus, passing strict_variables = true